### DISCUSSION TITLE
按照官方容器组更新后成了全新站点，数据都没了，怎么解决？

### DISCUSSION THREAD

**Discussion by leealu** [halo-dev/halo#3719] [Q&A]
### 是什么版本出现了此问题？

2.4.2

### 使用的什么数据库？

H2

### 使用的哪种方式部署？

Docker Compose

### 在线站点地址

blog.801100.tk

### 发生了什么？

从2.41更新到2.42.按照官方文档的容器更新，更新完成了新站点，文件夹里东西都在，这怎么恢复？

![photo_2023-04-07_22-41-25](https://user-images.githubusercontent.com/103651858/230627720-884d361d-219f-47a4-b5a0-f313fe8020dd.jpg)
![photo_2023-04-07_22-41-31](https://user-images.githubusercontent.com/103651858/230627740-b240e316-374c-48bf-a334-e99a259273d9.jpg)

**Comment by leealu** [comment]
我的yam部署的时候容器名写的是halo2·现在更新了没有变·还是halo2·这有影响吗？

**Comment by ruibaby** [comment]
根据你所说的按照官方文档的步骤升级没有复现，这样的描述没人能知道你具体做了什么。

**Reply by leealu** [reply]
> 根据你所说的按照官方文档的步骤升级没有复现，这样的描述没人能知道你具体做了什么。

我现在有备份的完整文件，我现在用docker-com去安装新的并且恢复数据我改怎么做？

# 修改为自己已有的 MySQL 配置
      - --spring.r2dbc.url=r2dbc:h2:file:///root/.halo2/db/halo-next?MODE=MySQL&DB_CLOSE_ON_EXIT=FALSE
      - --spring.r2dbc.username=root
      - --spring.r2dbc.password=
      - --spring.sql.init.platform=h2
是不是要这样去写数据库？halo-next?这里写完整的数据库名还是就这样带？
